index.html

```diff
@@ -5,7 +5,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <link href="favicon.ico" rel="shortcut icon" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="src/sass/style.css">
+    <!-- <link rel="stylesheet" href="src/sass/style.css"> -->
```

Comment:
don't be afraid to delete code insted of commenting out. most of the time you won't need it anymore